### PR TITLE
Testing / fix permissions

### PR DESCRIPTION
**Pull request by lapineige**
This reverts commit ae6795704864e061bf5a5133ffd4ddd375c676e2.

CSS is broken on new install, reverting before we find a solution for this.

### CODE REVIEW

**Comment by lapineige** [comment]
Oh, dammit, I can't merge without a review… @ericgaspar can you do it please ?

Then I'll investigate what broke the CSS in your PR… After a first look, I really have no idea…

**Comment by lapineige** [comment]
ping @ericgaspar :up: :)

**Comment by alexAubin** [comment]
Ugh this looks like a really brutal way to solve the issue ... Having a look at the changelog, I can't see anything remotely related to the CSS unable to being loaded ... It would really be better to investigate what's happening exactly, for example using Firefox's debugger ...

File: manifest.json [hunk 1]
```diff
@@ -4,49 +4,62 @@
...
-    },
   "license": "MIT",
   "maintainer": {
     "name": "lapineige et rafi59",
     "email": "rafi59_dev@srvmaison.fr.nf",
     "url": ""
   },
   "requirements": {
-    "yunohost": ">= 4.3.0"
+    "yunohost": ">= 3.8.1"
```
Comment (alexAubin) [inline manifest.json:18]: Ah in fact it's probably this: requiring yunohost higher than 4.2 implicitly triggers some permissions on /var/www/$app

We probably need to add some tweaks like `chown $app:www-data /var/www/$app` in the install script (and upgrade and restore) assuming it's not already there

**Comment by lapineige** [comment]
> It would really be better to investigate what's happening exactly, for example using Firefox's debugger ...

That's sure, but having not knowledge or any clue about what would cause the issue (i would never guessed the issue you're raising) and limited time for that, I thought the best solution to solve what was a big breaking bug (all new installations are broken) was that. That would not create any significant side effect and would temporarily solve the issue while we could investigate what went wrong…
To be clear, I'm not depreciating the quality of @ericgaspar pull request, and I couldn't even find any issue inside it. That was just the easiest and quickest way to solve a major breaking change (which appears to be a side effect) from my perspective.

File: manifest.json (hunk 1 above)
Comment (lapineige) [reply manifest.json:18]: The thing is, according to this report https://github.com/YunoHost-Apps/shaarli_ynh/issues/75#issuecomment-986056415 the is no permission issue, they are identical…

**Comment by lapineige** [comment]
And if we don't find a fix quickly, I would still prefer that option.

**Comment by alexAubin** [comment]
!testme

**Comment by alexAubin** [comment]
So i wasn't able to reproduce the "no CSS" issue while installing the current version (`0.12.1~ynh2`)

Nevertheless there's definitely something funky with the permissions, so I force-pushed some changes to try to fix them

Feel free to try installing from this branch with `yunohost app install https://github.com/YunoHost-Apps/shaarli_ynh/tree/testing`

**Comment by lapineige** [comment]
I'm reproducing the issue on a fresh install of that branch :(

**Comment by alexAubin** [comment]
>I'm reproducing the issue on a fresh install of that branch :(

tealc_indeed.gif ... me too, there was an issue with www-data not being able to access the corresponding assets

Should be fixed in previous commit

**Comment by lapineige** [comment]
I have a blank page now :sweat_smile:

**Comment by lapineige** [comment]
I don't know what to do, I'm not comfortable about leaving the app broken for a long time, yet I don't want to rush anyone to find a solution to fix it. Should we revert the PR ?

**Comment by alexAubin** [comment]
@lapineige : sorry for the delay ... the blank page issue should be fixed now, can you try again ?

**Comment by lapineige** [comment]
No need to apologies :)

This solves the issue ! :tada:

Great, thanks for your work :smiley:

**Comment by alexAubin** [comment]
Great ! Let's go then !

**Pull request merged**